<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabPiu</class>
 <widget class="QWidget" name="TabPiu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabPiu</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>691</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>П-18 М</string>
    </attribute>
    <widget class="QGroupBox" name="grBoP18M">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>641</height>
      </rect>
     </property>
     <property name="title">
      <string>наличие</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <widget class="QGroupBox" name="groupBox_199">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>551</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>точка стояния</string>
      </property>
      <widget class="PLineEdit" name="liEdPointStandingDistance">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="PLineEdit" name="liEdPointStandingAzimuth">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>50</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_161">
       <property name="geometry">
        <rect>
         <x>347</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>дальность:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_162">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>азимут:</string>
       </property>
      </widget>
      <widget class="PLineEdit" name="liEdLongitude">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_163">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>широта:</string>
       </property>
      </widget>
      <widget class="PLineEdit" name="liEdLatitude">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_164">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>долгота:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_227">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>30</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>склонение</string>
      </property>
      <widget class="QRadioButton" name="raBuGradientMagnetic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>магнитное</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="raBuGradientGeographic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>141</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>географич.</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>371</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>первичный канал</string>
      </property>
      <widget class="QGroupBox" name="groupBox_218">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>110</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>номер РЛС</string>
       </property>
       <widget class="QComboBox" name="coBoPrimarRlsNumber">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>06</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>07</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_202">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>название</string>
       </property>
       <widget class="PLineEdit" name="liEdPrimarName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_251">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>доворот</string>
       </property>
       <widget class="PLineEdit" name="liEdPrimTurn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_252">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>блокировка видео</string>
       </property>
       <widget class="PLineEdit" name="liEdPrimarLockVideo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_253">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>уровень 1</string>
       </property>
       <widget class="PLineEdit" name="liEdPrimLev1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_254">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>150</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>уровень 2</string>
       </property>
       <widget class="PLineEdit" name="liEdPrimLev2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_255">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>уровень 3</string>
       </property>
       <widget class="PLineEdit" name="liEdPrimLev3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grBoSecCh">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>371</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>вторичный канал1</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <widget class="QGroupBox" name="groupBox_220">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>110</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>номер АПОИ</string>
       </property>
       <widget class="QComboBox" name="coBoSecondApoiNumber">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>04</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_204">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>название</string>
       </property>
       <widget class="PLineEdit" name="liEdSecondName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_261">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>доворот</string>
       </property>
       <widget class="PLineEdit" name="liEdSecondTurn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_382">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>адрес IP РЛС</string>
       </property>
       <widget class="OLineEdit" name="liEdSecondIpRls">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_383">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>порт</string>
       </property>
       <widget class="OLineEdit" name="liEdSecondPortBase">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_384">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>адрес IP платы</string>
       </property>
       <widget class="OLineEdit" name="liEdSecondIpBoard">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_39">
    <attribute name="title">
     <string>Общие</string>
    </attribute>
    <widget class="QLabel" name="label_56">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Плата ПИУ</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_364">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>функциональное название платы</string>
     </property>
     <widget class="PLineEdit" name="liEdNameBoard">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_369">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>порт платы</string>
     </property>
     <widget class="OLineEdit" name="liEdPortBoard">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_370">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>склонение</string>
     </property>
     <widget class="OLineEdit" name="liEdSclonenie">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_371">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>адрес IP</string>
     </property>
     <widget class="OLineEdit" name="liEdIp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_372">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>базовый порт</string>
     </property>
     <widget class="OLineEdit" name="liEdPortBase">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_67">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>разрядность угла</string>
     </property>
     <widget class="QComboBox" name="comboBoxRankAngle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>10 разрядов</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11 разрядов</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12 разрядов</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PLineEdit</class>
   <extends>QLineEdit</extends>
   <header>plineedit.h</header>
  </customwidget>
  <customwidget>
   <class>OLineEdit</class>
   <extends>QLineEdit</extends>
   <header>olineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>grBoP18M</tabstop>
  <tabstop>liEdLatitude</tabstop>
  <tabstop>liEdLongitude</tabstop>
  <tabstop>liEdPointStandingDistance</tabstop>
  <tabstop>liEdPointStandingAzimuth</tabstop>
  <tabstop>raBuGradientMagnetic</tabstop>
  <tabstop>raBuGradientGeographic</tabstop>
  <tabstop>liEdPrimarName</tabstop>
  <tabstop>liEdPrimTurn</tabstop>
  <tabstop>liEdPrimarLockVideo</tabstop>
  <tabstop>liEdPrimLev1</tabstop>
  <tabstop>liEdPrimLev2</tabstop>
  <tabstop>liEdPrimLev3</tabstop>
  <tabstop>liEdSecondName</tabstop>
  <tabstop>liEdSecondTurn</tabstop>
  <tabstop>liEdSecondIpRls</tabstop>
  <tabstop>liEdSecondPortBase</tabstop>
  <tabstop>liEdNameBoard</tabstop>
  <tabstop>liEdSclonenie</tabstop>
  <tabstop>liEdIp</tabstop>
  <tabstop>liEdPortBase</tabstop>
  <tabstop>liEdPortBoard</tabstop>
  <tabstop>comboBoxRankAngle</tabstop>
  <tabstop>tabWidget_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
